conda-recipe/run_test.sh

```diff
@@ -0,0 +1 @@
+nosetests -v metadatastore --cover-package metadatastore --with-coverage
```

Comment:
Oh this is what is was:

We will test the conda build on Travis. We will test the conda build in the git repo. We will not include the test script with the recipes on conda-prescriptions. The logic being that our tagged versions have been thoroughly tested, but if you are building a conda package from a non tagged version it is better to test it during the build.

Also it is very easy to skip the testing by simply deleting the run_tests.sh file